<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icon.png</normaloff>:/Icons/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QLabel" name="image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuTransform">
     <property name="title">
      <string>Transform</string>
     </property>
     <addaction name="actionBlackWhite"/>
    </widget>
    <addaction name="actionOpenFile"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionResize"/>
    <addaction name="menuTransform"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFolder">
    <property name="title">
     <string>Folder</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuReload">
     <property name="title">
      <string>Reload when file changes</string>
     </property>
     <addaction name="actionReloadAuto"/>
     <addaction name="actionReloadAsk"/>
     <addaction name="actionReloadNo"/>
    </widget>
    <addaction name="menuReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFolder"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionRotateLeft"/>
   <addaction name="actionRotateRight"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename all files according to date</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Right.png</normaloff>:/Icons/Right.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next (Right)</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Left.png</normaloff>:/Icons/Left.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous (Left)</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/RotateLeft.png</normaloff>:/Icons/RotateLeft.png</iconset>
   </property>
   <property name="text">
    <string>Rotate left</string>
   </property>
   <property name="toolTip">
    <string>Rotate left (Ctrl+Left)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/RotateRight.png</normaloff>:/Icons/RotateRight.png</iconset>
   </property>
   <property name="text">
    <string>Rotate right</string>
   </property>
   <property name="toolTip">
    <string>Rotate right (Ctrl+Right)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Delete.png</normaloff>:/Icons/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete (Del)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionBlackWhite">
   <property name="text">
    <string>black and white</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionReloadAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionReloadNo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </action>
  <action name="actionReloadAsk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ask</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
